custom_components/garmin_connect/__init__.py

```
@@ -1,91 +1,90 @@
                     CONF_TOKEN: tokens,
                 }
 
-                # Update the config entry
                 hass.config_entries.async_update_entry(entry, data=new_data)
-
-                _LOGGER.info(
-                    "Successfully migrated Garmin Connect config entry")
+                _LOGGER.info("Migration successful")
                 return True
 
             except Exception as err:  # pylint: disable=broad-except
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)